org.eclipse.gef/src/org/eclipse/gef/ui/actions/SelectionAction.java

```diff
@@ -65,26 +65,41 @@ public void dispose() {
 	}
 
 	/**
-	 * Returns a <code>List</code> containing the currently selected objects.
+	 * Returns a {@code List<EditPart} containing the currently selected objects.
+	 * The {@code set} method is supported but the {@code add}, {@code addAll},
+	 * {@code remove} {@code removeAll} methods are not. Changes to this list modify
+	 * the internal {@link #selection}.
 	 *
 	 * @return A List containing the currently selected objects.
 	 */
-	protected List<Object> getSelectedObjects() {
+	protected List<?> getSelectedObjects() {
```

Comment:
This is an API breaking change and break BIRT's build:

https://github.com/eclipse-birt/birt/pull/1972#issuecomment-2482704548

Please revert it.